### DISCUSSION TITLE
enable  free security features (KIBANA)

### DISCUSSION THREAD

**Discussion by jotaah123** [telekom-security/tpotce#1025] [Q&A]
Hello Github! 

I need help when i open the Kibana , the first message is: 
![imagen](https://user-images.githubusercontent.com/39537659/155389807-c07d96d9-1372-4b3c-bde0-b61130ba8857.png)

but idk how to do on dockers the hardening of kibana

**Comment by t3chn0m4g3** [comment]
Please see #889, with the secure reverse proxy in place there is no immediate need to enable Kibana's security features.
